### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kensonjohnson**
- Added badges for technologies used.
- Added small description with link to relativepath.tech.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,13 +23,13 @@ A simple setup using:
...
 
 ```bash
 cd ./relativepath.tech
 ```
 
-3. Run the `setup.sh` bash script to make sure your system has necessary requirements:
+3. Run the `setup.sh` bash script to install requirements:
 
 ```bash
 bash setup.sh
```
Comment (stephenlauck) [inline README.md]: What if we rename this file to local.sh, then later if we have like local-darwin.sh or local-powershell.sh things will make sense? I think naming the script based on the context it runs or the context+job it does is maybe something that could work.

**Review by stephenlauck — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mflorida) [reply README.md]: > What if we rename this file to local.sh, then later if we have like local-darwin.sh or local-powershell.sh things will make sense? I think naming the script based on the context it runs or the context+job it does is maybe something that could work.

How about passing a flag or argument like `setup.sh --macos`?

Comment (stephenlauck) [reply README.md]: In my experience it's easier to have a script per platform than a complex script that tries to do each platform.

Comment (kensonjohnson) [reply README.md]: This change has been made.

**Comment by mflorida** [comment]
@stephenlauck Do you want to add me as a reviewer? (And as a contributor to the project in general - or are we handling pull requests from forks?)

**Review by stephenlauck — APPROVED** [review]

**Review by mflorida — APPROVED** [review]
😎👍

**Review by mflorida — APPROVED** [review]

**Pull request merged**